lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -369,6 +372,13 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
             : null;
     }
 
+    private validateSpecialCharacters(tagNameControl: FormControl<string>): TagNameControlErrors | null {
+        const specialSymbolsRegex: RegExp = /[':"\\|<>\/?]/;
+        return tagNameControl.value.length && specialSymbolsRegex.test(tagNameControl.value)
```

Comment:
Maybe a null safety check on tagNameControl.value?

Reply:
There is default value for control to avoid checking null in whole file for every place: 
<img width="772" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/69338078-be25-42c5-a3cd-d6d566ae1ccd">


So there should not be a case with null I guess